frontend/src/components/debug/UserDebug.vue

```diff
@@ -58,7 +58,7 @@ const updateUser = () => {
 
 const deleteUser = (id: number) => {
   Delete(baseUrl + '/' + id.toString()).then(res => {
-    if (res.status == 204) {
+    if (res.status == 200) {
```

Comment:
Seems to return `200` instead of `204` when resource is deleted

Reply:
Yeah, the API response on DELETE request is 200 OK, as we get what we expect (404 if ressource not found).
To return a 204 No Content, I need to throw an error (what I did before), or maybe @HttpCode() that will be override by the throw 404?